css-paint-api/Overview.bs

```diff
@@ -138,13 +138,22 @@ interface PaintWorkletGlobalScope : WorkletGlobalScope {
 };
 </pre>
 
+The {{PaintRenderingContext2DSettings}} allows the settings for the rendering context associated
+with the paint canvas. Right now, the {{PaintRenderingContext2DSettings}} has a boolean flag
```

Comment:
I think it would be more clear to have a bulleted list of what each value means. Also, perhaps instead of implying that we may add more settings we should say this provides a supported subset of the canvas context 2d settings (ideally with a link)

Reply:
Changed, please review the new commit.